spring-cloud-deployer-resource-maven/src/main/java/org/springframework/cloud/deployer/resource/maven/MavenProperties.java

```diff
@@ -197,6 +211,11 @@ public RemoteRepository(final String url) {
 			this.url = url;
 		}
 
+		public RemoteRepository(final String url, final Authentication auth) {
```

Comment:
I do but more just for testing convenience. See [here](https://github.com/donovanmuller/spring-cloud-deployer-openshift/blob/master/src/test/java/org/springframework/cloud/deployer/spi/openshift/factories/BuildConfigFactoryTest.java#L95)